src/types/nitro.ts

```diff
@@ -31,11 +31,21 @@ export type NitroDynamicConfig = Pick<
   "runtimeConfig" | "routeRules"
 >;
 
-export interface NitroRuntimeConfig {
-  app: {
-    baseURL: string;
+export interface NitroRuntimeConfigApp {
+  baseURL: string;
+  [key: string]: unknown;
+}
+
+export interface NitroRuntimeConfigNitro {
```

Comment:
Do you think we need to augment this too?

Reply:
No, I think it is just good for internal type safety. We could even type it in a way that we _forbid_ augmenting it if you want, but leave possibility open for ourselves later.

Reply:
Yeah for now let’s keep it inlined I don’t see a reason to externally extend nitro namespace